workspace/apps/pidp/src/assets/snowplow.js

```diff
@@ -0,0 +1,38 @@
+// <!-- Snowplow starts plowing - Standalone vE.2.14.0 -->
+(function (p, l, o, w, i, n, g) {
+  if (!p[i]) {
+    p.GlobalSnowplowNamespace = p.GlobalSnowplowNamespace || [];
+    p.GlobalSnowplowNamespace.push(i);
+    p[i] = function () {
+      (p[i].q = p[i].q || []).push(arguments);
+    };
+    p[i].q = p[i].q || [];
+    n = l.createElement(o);
+    g = l.getElementsByTagName(o)[0];
+    n.async = 1;
+    n.src = w;
+    g.parentNode.insertBefore(n, g);
+  }
+})(
+  window,
+  document,
+  'script',
+  'https://www2.gov.bc.ca/StaticWebResources/static/sp/sp-2-14-0.js',
+  'snowplow',
+);
+var collector = 'spm.apps.gov.bc.ca';
+window.snowplow('newTracker', 'rt', collector, {
+  appId: 'Snowplow_standalone',
+  cookieLifetime: 86400 * 548,
```

Comment:
why specifically the 548?
Can we keep this number in any config and use it here ?. Later if we want to change the cookieLifetime then simply changing the config without changing this snowplow.js file will work.

Reply:
This is part of testing environment code that we were asked to add as is to the application. It is still unclear how/if snowplow tracking will be utilized.